apps/web/src/components/Sidebar/DocumentTree/index.tsx

```diff
@@ -54,31 +83,56 @@ function TreeNode({ node, level = 0 }: { node: Node; level?: number }) {
             {node.doc.documentType === 'folder' ? (
               <>
                 <p>{node.doc.name}</p>
-                <CreateNode parentId={node.doc.id} />
+                <CreateNode commit={commit} parentId={node.doc.id} />
               </>
             ) : (
               <p className='font-bold'>{node.doc.name}</p>
             )}
           </div>
         )}
         {node.children.map((node, idx) => (
-          <TreeNode key={idx} node={node} level={level + 1} />
+          <TreeNode key={idx} commit={commit} node={node} level={level + 1} />
         ))}
       </div>
     </div>
   )
 }
 
 export default function DocumentTree({
+  commits: serverCommits,
   documents: serverDocuments,
 }: {
+  commits: Commit[]
   documents: DocumentVersion[]
 }) {
-  const { documents } = useDocumentVersions(
-    { staged: true },
+  const [commit, setCommit] = useState(serverCommits[0]!.uuid)
+  const { data: documents } = useDocumentVersions(
+    { commitUuid: commit },
     { fallbackData: serverDocuments },
   )
+  const { data: commits, create } = useCommits(null, {
+    fallbackData: serverCommits,
+  })
+
   const rootNode = useTree({ documents })
 
-  return <TreeNode node={rootNode} />
+  return (
+    <div className='flex flex-col gap-12 pt-4 pl-4'>
+      <button onClick={() => create()}>Create commit</button>
+      <select onChange={(ev) => setCommit(ev.currentTarget.value)}>
+        {commits.map((c) => (
+          <option key={c.uuid} value={c.uuid}>
+            {c.uuid}
+          </option>
+        ))}
+      </select>
+      <div className='flex flex-col gap-4'>
+        <div className='flex flex-row align-items justify-between'>
+          <h2 className='font-bold'>Create document</h2>
+          <CreateNode commit={commit} />
+        </div>
+        <TreeNode commit={commit} node={rootNode} />
+      </div>
+    </div>
+  )
 }
```

Comment:
most of this ui will probably get removed so don't pay too much attention to it.

Reply:
My next task is to do the sidebar styles.